pkg/webhook/common.go

```diff
@@ -70,6 +70,18 @@ func (h *webhookHandler) getConfig(ctx context.Context) (*v1alpha1.Config, error
 	return cfg, h.client.Get(ctx, keys.Config, cfg)
 }
 
+// validateGatekeeperResources returns whether an issue is user error (vs internal) and any errors
+// validating internal resources
+func (h *webhookHandler) isGatekeeperResource(ctx context.Context, req admission.Request) bool {
+	if req.AdmissionRequest.Kind.Group == "templates.gatekeeper.sh" ||
```

Comment:
add `config` and `status`?

Reply:
This is actually a change in #881, on which this PR depends (and is rebased).
Updated this in #881